docs/extend/plugins_graphdriver.md

```diff
@@ -47,20 +81,22 @@ Respond with a non-empty string error if an error occurred.
 ### /GraphDriver.Create
 
 **Request**:
-```
+```json
 {
   "ID": "46fe8644f2572fd1e505364f7581e0c9dbc7f14640bd1fb6ce97714fb6fc5187",
-  "Parent": "2cd9c322cb78a55e8212aa3ea8425a4180236d7106938ec921d0935a4b8ca142"
-  "MountLabel": ""
+  "Parent": "2cd9c322cb78a55e8212aa3ea8425a4180236d7106938ec921d0935a4b8ca142",
+  "MountLabel": "",
+  "StorageOpt": {}
 }
 ```
 
 Create a new, empty, read-only filesystem layer with the specified
```

Comment:
Maybe something like:

If `Parent` is an empty string, there is no parent layer.

`StoreOpt` is a map of strings which indicate storage options.